internal/driver/glfw/window_wasm.go

```diff
@@ -474,7 +475,11 @@ func (w *window) rescaleOnMain() {
 		scale.ToFyneCoordinate(w.canvas, w.width),
 		scale.ToFyneCoordinate(w.canvas, w.height))
 	w.canvas.Resize(scaledFull)
-	return
+
+	// Ensure textures re-rasterize at the new scale
+	cache.DeleteTextTexturesFor(w.canvas)
+	w.canvas.content.Refresh()
+	//return
```

Comment:
I think committing commented out code goes against our contribution guide. I guess that applies to the code below as well?

Reply:
Done. I suppose the git history will always have the commented-out lines if we ever need to switch back (I have no idea why it was commented-out in the first place)